libs/features/georoutingv2/georoutingv2.js

```diff
@@ -281,12 +283,19 @@ export default async function loadGeoRouting(
   loadBlock = loadBlockFunc;
   loadStyle = loadStyleFunc;
 
-  const resp = await fetch(`${config.contentRoot ?? ''}/georoutingv2.json`);
+  let resp = await fetch(`${config.contentRoot ?? ''}/georoutingv2.json`);
   if (!resp.ok) {
-    // eslint-disable-next-line import/no-cycle
-    const { default: loadGeoRoutingOld } = await import('../georouting/georouting.js');
-    loadGeoRoutingOld(config, createTag, getMetadata);
-    return;
+    resp = await fetch(`${config.contentRoot ?? ''}/georouting.json`);
```

Comment:
I think we're mixing scope here and this will eventually be hard to remove when we fully deprecate the old geo-routing feature.

Reply:
To provide backward compatibility, we still need to ensure that fetch(${config.contentRoot ?? ''}/georoutingv2.json) doesn't exist before we go ahead and load the federal georoutingv2 file.
I am not seeing any other way to handle this case without introducing additional configuration from consumer. However, I am open to suggestions.

Reply:
We could always configure a project and say pick up the georouting from federal via metadata.